fireblocks_sdk/sdk.py

```diff
@@ -1957,7 +1981,7 @@ def get_vault_balance_by_asset(self, asset_id=None):
         return self._get_request(url)
 
     def create_raw_transaction(
-        self, raw_message, source=None, asset_id=None, note=None
+        self, raw_message: RawMessage, source=None, asset_id=None, note=None
```

Comment:
this would break current users that are using typing, I wouldn't introduce this.